functiondefextractor/extractor.log

```diff
@@ -0,0 +1,30 @@
+2020-07-31 10:26:34,002 Input repository path validated successfully
+2020-07-31 10:26:34,002 Input report folder path validated successfully
+2020-07-31 10:26:34,005 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\CerberusTest.java
+2020-07-31 10:26:34,322 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\CerberusTest.java
+2020-07-31 10:26:34,322 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\HelloController.java
+2020-07-31 10:26:34,640 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\HelloController.java
+2020-07-31 10:26:34,640 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\python_annot_file.py
+2020-07-31 10:26:34,863 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\python_annot_file.py
+2020-07-31 10:26:34,864 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\python_file.py
+2020-07-31 10:26:35,077 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\python_file.py
+2020-07-31 10:26:35,078 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\test_c.c
```

Comment:
you could consider removing the file from repo as its execution log!

Reply:
Corrected